honeybadgermpc/protocols/crypto/boldyreva.py

```diff
@@ -173,7 +173,10 @@ def __setstate__(self, d):
 def dealer(players=10, k=5, seed=None):
     """ """
     # Random polynomial coefficients
-    a = group.random(ZR, count=k, seed=seed)
+    if seed is not None:
+        a = [group.random(ZR, seed=seed+i) for i in range(k)]
```

Comment:
can we file a bug report to charm about this

Reply:
I will get rid of charm eventually and replace it with a suitable `betterpairing` version. I will still file a bug on them. I have opened #217 (assigned to me) to track this.